### DISCUSSION TITLE
Incomplete Sync

### DISCUSSION THREAD

**Discussion by TechnicallyBob202** [gilesknap/gphotos-sync#406] [Q&A] [answered]
I got everything working to sync to my Synology by using the docker container, but it is only syncing back to 2021, and I have pictures in there for nearly 10 years.

Is this a Google Photos API rate limit?  Will it eventually go all the way back?

**Comment by TechnicallyBob202** [comment]
OK.  so, I figured it out.  the OAuth token is no longer working.  What's up with that?

**Comment by TechnicallyBob202** [comment, marked as answer]
OK... and now that i ensured that the refresh token is working, it looks like it did download all of the files, but Synology Photos just hasn't indexed them all!

**Comment by gilesknap** [comment]
Maybe Synology needs a bit more time? :-) I guess we can close this ticket?

**Comment by gilesknap** [comment]
Oh its a discussion! marking as answered ...
